ci/build.sbt

```diff
@@ -1,2 +1,4 @@
 addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.14.2")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "1.0.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.0") // scala-steward:off
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0") // scala-steward:off
```

Comment:
Since the CI plugin is trying not to be too prescriptive, and we don't use the JS/native plugins except to check for their presence on a project, these versions should be sufficient.

Hopefully this won't interfere with our other updates.

Reply:
Does the Native plugin have have the non-breaking track record of Scala.js?  What is the risk of this dependency to all the people who don't aspire to support Native, which I think is still most Typelevel projects?

Reply:
All fair points, I'm not familiar enough with native but it's still much more in development. In any case, I ended up removing it because my plugin detection idea wasn't working. See https://github.com/typelevel/sbt-typelevel/pull/43#discussion_r777820570.